projects/transformers/requirements.txt

```diff
@@ -20,6 +20,9 @@
 #
 # ------------------------------------------------------------------------------
 # Run pip install -r requirements.txt to reproduce the currently tested environment
-# Last update: 2/17/2021
--e git+https://github.com/huggingface/transformers@e94d63f6cbf5efe288e41d9840a96a5857090617#egg=transformers
--e git+https://github.com/huggingface/datasets@58555b2734b8a18d70b5a2418129f26fedd96a39#egg=datasets
\ No newline at end of file
+# Last update: 3/29/2021
+
+git+git:https://github.com/huggingface/transformers@master#egg=transformers
```

Comment:
The PR description mentions `4.5.0.dev0` but this code will install the latest version from `master`. If the goal is stability then you should update the commit to `4.5.0.dev0` instead of plain `master` branch

Reply:
I am fine either the way, but as reminder we can't fix transformers without fixing datasets and tokenizers - we had issues in the past with mismatch, and they don't guarantee the latest versions in pip are synced (they do guarantee the latest versions from source are synced)

Reply:
We're balancing two motivations: hedging risk against errors caused by misaligned dependencies and having the latest features of the Huggingface repos. We've been prioritizing the latter for a faster pace of development (as we're actively waiting for planned upcoming features). I'll keep these dependencies as is, but we can restrict them more as needed down the line.

Reply:
Just keep in mind `master` is a moving target and the code installed will depend on the day you call `pip install`